xpad.c

```diff
@@ -101,7 +102,8 @@
 #define XTYPE_XBOX360     1
 #define XTYPE_XBOX360W    2
 #define XTYPE_XBOXONE     3
-#define XTYPE_UNKNOWN     4
+#define XTYPE_XBOXONEWHL  4
```

Comment:
do not introduce a new XTYPE_ - if anything this should be a wheel mapping for the XTYPE_XBOXONE